### DISCUSSION TITLE
Reviewers populated from CODEOWNERS even if reviewersFromCodeOwners is disabled

### DISCUSSION THREAD

**Discussion by mdantonio** [renovatebot/renovate#19280] [Request Help] [answered]
We have Renovate configured on a private repository, so I will add a portion of the configuration that could be relevant:

```
{
  $schema: 'https://docs.renovatebot.com/renovate-schema.json',
  extends: [
    'config:base',
    ':combinePatchMinorReleases',
    'schedule:weekly',
  ],
  packageRules: [
    {
      matchManagers: [
        'pip_requirements',
        'pyenv',
        'gomod',
      ],
      reviewers: [
        'team:backend',
      ],
    },
```
We also have a `CODEOWNERS` file that contains:

```
/api/infra/    @myorg/infra
```

and `reviewersFromCodeOwners` disabled (tried both to not define it since it is false by default, but also to explicitly set it to false)

What we are observing is that dependencies located in any folder with no match with our CODEOWNERS file are correctly assigned to the backend team (`reviewers: ['team:backend',]`), but dependencies located in folders with a match in CODEOWNERS (like /api/infra/) are assigned for review to the group defined there and `reviewers` (and `additionalReviewers` as well) are then ignored.

So it looks like `CODEOWNERS` takes the priority respect to `reviewers`/`additionalReviewers` even if `reviewersFromCodeOwners` is disabled.
Our goal is to only leverage on the Renovate configuration and completely ignore `CODEOWNERS` entries.

Is there something I misunderstood from the configuration or is it a bug?

**Comment by rarkins** [comment, marked as answer]
Which platform?

**Reply by mdantonio** [reply]
github

**Reply by rarkins** [reply]
GitHub populates the reviewers automatically in this case - it's not Renovate doing it. Ref: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners

**Reply by mdantonio** [reply]
Oh, I got it.
So in this case `reviewers` option is simply ignored, because reviewers are already assigned by GitHub through CODEOWNERS?
And`additionalReviewers` is also ignored in this case? Or could I leverage on that? If both are ignored, then there is no way to bypass CODEOWNERS, except by removing it?

**Reply by rarkins** [reply]
Renovate will assign whatever reviewers/additionalReviewers you set. GitHub will then assign reviewers based from CODEOWNERS if they're not already on the list.

**Reply by mdantonio** [reply]
This is not the case, it seems.
As reported, we are getting reviewers assigned by Renovate only when there are no matches from CODEOWNERS, so the assignments from GitHub and Renovate seem to be mutually exclusive. In particular Renovate is not assigning reviewers if already assigned by GitHub

**Reply by rarkins** [reply]
Please create a reproduction repo on github.com so we can take a look

**Reply by mdantonio** [reply]
Thank you for the suggestion.
Tried with a fresh repository trying to reproduce a similar setup and I can confirm that Renovate works as expected, being able to combine reviewers from CODEOWNERS, reviewers and additionalReviewers. 🙌 
But when I switched the repository from public to private, Renovate started to skip the review assignment at all. Any possible permission issue?
